<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChannelGUI</class>
 <widget class="QFrame" name="ChannelGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>761</width>
    <height>371</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ChannelGUI</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QFrame
{
	border-width: 5px;
	border-style: inset;
	border-radius: 20px;
	background: url(:/images/channels.png);
}

QLabel
{
	color: white;
	border: none;
	background:none;
}

QLabel#labelClipDuration,
QLabel#labelClipName,
QLabel#labelClipRemaining
{
	border-width: 2px;
	border-radius: 10px;
	border-style: solid;
	border-color: grey;
	color: black;
	background:white;
}

QPushButton#pushButtonTake,
QPushButton#pushButtonDrop,
QPushButton#pushButtonRecue,
QPushButton#pushButtonLoop
{
	background-color: lightGrey;
	border-style: outset;
	border-width: 10px;
	border-color: grey;
	border-radius: 20px;
}
QPushButton#pushButtonTake:pressed,
QPushButton#pushButtonDrop:pressed,
QPushButton#pushButtonRecue:pressed,
QPushButton#pushButtonLoop:pressed
{
	border-style: inset;
}

QPushButton#pushButtonLoop:checked
{
	background-color: gold;
	border-color: darkGoldenrod;
}

QFrame#videoFrame
{
	border-width: 2px;
	border-radius: 0px;
	background: none;
	border-style: solid;
	border-color: grey;
}

QLabel#labelNoRef
{
	color: red !important;
}</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <widget class="QPushButton" name="pushButtonTake">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>20</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>26</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>TAKE</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelChannel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>45</y>
     <width>61</width>
     <height>281</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>19</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>C
h
a
n
n
e
l</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonLoop">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>230</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>26</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>LOOP</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonDrop">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>90</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>26</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>DROP</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonRecue">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>160</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>26</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>RECUE</string>
   </property>
  </widget>
  <widget class="QVUMeter" name="audioMeter" native="true">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>70</y>
     <width>81</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelClipRemaining">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>42</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>00:00:00:00</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QFrame" name="videoFrame">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>70</y>
     <width>388</width>
     <height>220</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="PreviewerRGB" name="preview" native="true">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>2</y>
      <width>384</width>
      <height>216</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="labelClipName">
   <property name="geometry">
    <rect>
     <x>89</x>
     <y>14</y>
     <width>462</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>No Clip Loaded</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QSlider" name="sliderSeek">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>293</y>
     <width>388</width>
     <height>19</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QSlider::groove:horizontal {
border: 1px solid #bbb;
background: white;
height: 10px;
border-radius: 4px;
}

QSlider::sub-page:horizontal {
background: qlineargradient(x1: 0, y1: 0,    x2: 0, y2: 1,
    stop: 0 #66e, stop: 1 #bbf);
background: qlineargradient(x1: 0, y1: 0.2, x2: 1, y2: 1,
    stop: 0 #bbf, stop: 1 #55f);
border: 1px solid #777;
height: 10px;
border-radius: 4px;
}

QSlider::add-page:horizontal {
background: #fff;
border: 1px solid #777;
height: 10px;
border-radius: 4px;
}

QSlider::handle:horizontal {
background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
    stop:0 #eee, stop:1 #ccc);
border: 1px solid #777;
width: 13px;
margin-top: -2px;
margin-bottom: -2px;
border-radius: 4px;
}

QSlider::handle:horizontal:hover {
background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
    stop:0 #fff, stop:1 #ddd);
border: 1px solid #444;
border-radius: 4px;
}

QSlider::sub-page:horizontal:disabled {
background: #bbb;
border-color: #999;
}

QSlider::add-page:horizontal:disabled {
background: #eee;
border-color: #999;
}

QSlider::handle:horizontal:disabled {
background: #eee;
border: 1px solid #aaa;
border-radius: 4px;
}</string>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonPreviousFrame">
   <property name="geometry">
    <rect>
     <x>185</x>
     <y>316</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	border: none;
	background: url(:/images/buttons/previousframe.png);
}

QPushButton:pressed
{
	background: url(:/images/buttons/previousframe_pressed.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonForward">
   <property name="geometry">
    <rect>
     <x>395</x>
     <y>316</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	border: none;
	background: url(:/images/buttons/forward.png);
}

QPushButton:pressed
{
	background: url(:/images/buttons/forward_pressed.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonPlayPause">
   <property name="geometry">
    <rect>
     <x>255</x>
     <y>316</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	border: none;
	background: url(:/images/buttons/play.png);
}

QPushButton:pressed
{
	background: url(:/images/buttons/play_pressed.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonNextFrame">
   <property name="geometry">
    <rect>
     <x>325</x>
     <y>316</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	border: none;
	background: url(:/images/buttons/nextframe.png);
}

QPushButton:pressed
{
	background: url(:/images/buttons/nextframe_pressed.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonReverse">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>316</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	border: none;
	background: url(:/images/buttons/reverse.png);
}

QPushButton:pressed
{
	background: url(:/images/buttons/reverse_pressed.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelSpeed">
   <property name="geometry">
    <rect>
     <x>615</x>
     <y>310</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>32</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>1x</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelClipDuration">
   <property name="geometry">
    <rect>
     <x>89</x>
     <y>42</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>00:00:00:00</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelNoRef">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>315</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>32</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>No Ref</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVUMeter</class>
   <extends>QWidget</extends>
   <header>qvumeter.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PreviewerRGB</class>
   <extends>QWidget</extends>
   <header>previewerrgb.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
